packages/explorerkit-server/src/server.ts

```diff
@@ -343,8 +358,35 @@ async function decodeInstruction(instruction: Instruction): Promise<Instruction>
   };
 }
 
-function postProcessDecodedInstruction(decodedInstructionData: any, decodedInstructionDataWithTypes: any): any {
-  if (decodedInstructionData === null || decodedInstructionDataWithTypes === null) {
```

Comment:
This change fixes the following error in explorer-kit prod logs:
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at postProcessDecodedInstruction (/usr/src/app/packages/explorerkit-server/dist/index.js:275:10)
    at decodeInstruction (/usr/src/app/packages/explorerkit-server/dist/index.js:259:39)
    at /usr/src/app/packages/explorerkit-server/dist/index.js:198:48